### PR TITLE
Enable automation for YaST in Container in ALP

### PR DESCRIPTION
**Pull request by skliu**
Enable automation for YaST in Container in ALP

**Ticket:**

- https://progress.opensuse.org/issues/128117

**Verification run:**
- [Development ALP Micro](https://openqa.opensuse.org/tests/overview?distri=alp&version=micro-0.1&build=3.5&groupid=110)
- [Development ALP Bedrock](https://openqa.opensuse.org/tests/overview?distri=alp&version=bedrock-0.1&build=5.1&groupid=111)

**Related PR:**
- [# 318](https://github.com/os-autoinst/opensuse-jobgroups/pull/318)

### CODE REVIEW

File: schedule/alp/alp_yast.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: alp_yast
+description: >
+    Maintainer: qe-core@suse.de
```
Comment (jknphy) [inline schedule/alp/alp_yast.yaml:4]: not our squad

Comment (skliu) [reply schedule/alp/alp_yast.yaml]: Fixed.

**Comment by lemon-suse** [comment]
No VRs available?

**Comment by skliu** [comment]
> No VRs available?

Is my command line correct?

d505:~ # `openqa-cli api --apikey xx --apisecret xx --osd --pretty -X POST isos DISTRI=alp VERSION=micro-0.1 FLAVOR=Default ARCH=x86_64 BUILD=3.5 _GROUP_ID=110 TEST="yast_1" PUBLISH_HDD_1=alp-micro-0.1-x86_64-3.5-textmode@uefi.qcow2 HDD_2=ignition.qcow2 `
403 Forbidden
{
   "error" : "Not authorized",
   "error_status" : 403
}

**Comment by skliu** [comment]
> No VRs available?

https://openqa.opensuse.org/tests/3301988, the VR failed, I am fighting on it.

File: schedule/alp/alp_yast.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: alp_yast
+description: >
+    Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
```
Comment (lemon-suse) [inline schedule/alp/alp_yast.yaml:4]: move this line under the real description is better, WDYT?

File: schedule/alp/alp_yast.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    Enable automation for YaST
```
Comment (lemon-suse) [inline schedule/alp/alp_yast.yaml:5]: change to 'Enable YaST automation test for ALP'?

File: schedule/alp/alp_yast.yaml (hunk 2 above)
Comment (skliu) [reply schedule/alp/alp_yast.yaml]: Deleted it as other schedule files.

File: schedule/alp/alp_yast.yaml (hunk 3 above)
Comment (skliu) [reply schedule/alp/alp_yast.yaml]: Done.

**Pull request merged**